### DISCUSSION TITLE
MapView clipping of model below "ground"

### DISCUSSION THREAD

**Discussion by sidrubs** [visgl/deck.gl#7236] [Q&A] [answered]
We have a few models that are above and below the ground. When using MapView with a pitch one can see the entire model just fine. However, the closer you approach a pitch of zero (top-down), the parts of the model below "ground" are not rendered. This is shown in the screenshots below, and you can see an example in [this](https://codepen.io/srubsmodelar/pen/PoewVjw) CodePen, when you drag the view towards a pitch of 0.

Entire model visible:
![image](https://user-images.githubusercontent.com/52378223/188484118-e242d6b4-732d-412c-b6ff-c3ff65422580.png)

Floor of model clipped when pitch approaches 0:
![image](https://user-images.githubusercontent.com/52378223/188484215-274a6a6e-0a2a-457e-8636-e57c2c45f661.png)

Is it possible to make the MapView's "ground" transparent?

**Comment by Pessimistress** [comment, marked as answer]
Try [farZMultiplier](https://deck.gl/docs/api-reference/core/map-view#farzmultiplier)

**Reply by sidrubs** [reply]
Thank you @Pessimistress that works.
